tensorflow/training/docker/2.6/py3/Dockerfile.cpu

```diff
@@ -142,26 +142,25 @@ RUN ${PIP} install --no-cache-dir -U \
     python-dateutil==2.8.1 \
     "pyYAML>=5.4,<5.5" \
     requests==2.24.0 \
+    "awscli<2" \
     mpi4py==3.0.3 \
     "sagemaker>=2,<3" \
-    "botocore>1.0,<2.0" \
-    "boto3>1.0,<2.0" \
-    "awscli<2" \
     sagemaker-experiments==0.* \
     "sagemaker-tensorflow>=2.6,<2.7" \
     "sagemaker-tensorflow-training>=20" \
     "shap>=0.39,<1" \
+    "boto3>1.0,<2.0" \
     "sasl<1" \
     "thrift_sasl<1" \
     "PyHive<1" \
     "sparkmagic<1" \
     "sagemaker-studio-sparkmagic-lib<1" \
-    "sagemaker-studio-analytics-extension==0.0.2" \
+    "sagemaker-studio-analytics-extension<1" \
     # Let's install TensorFlow separately in the end to avoid
     # the library version to be overwritten
  && ${PIP} install --no-cache-dir -U \
     ${TF_URL} \
-    tensorflow-io==0.21.0 \
+    tensorflow-io \
```

Comment:
Why unpin this version?

Reply:
I rebased from master and it got changed. will fix it